packages/ERTP/src/issuerKit.js

```diff
@@ -24,32 +24,29 @@ import './types.js';
  *
  *  `displayInfo` gives information to the UI on how to display the amount.
  *
- * @param {string} allegedName
- * @param {K} [assetKind=AssetKind.NAT]
- * @param {AdditionalDisplayInfo} [displayInfo={}]
+ * @param {MapStore<string,any>} issuerBaggage
```

Comment:
if we can't make an explicit type, it's safer to use `unknown` so the uses of it don't think they're type checking when they're not
```suggestion
 * @param {MapStore<string, unknown>} issuerBaggage
```

given that this MapStore must have particular keys and non-homogenous values, I'd like to consider having a type such as `Baggage<T>` that encodes the mapping. It could still be accessed by `baggage.get(key)` instead of `baggage[key]` so no runtime change needed. That's out of scope of this PR but I'm happy to supply it now or later.

Though as I read down, I think it may be better to change this signature to take an object. Will describe closer.

Reply:
All of the Baggages are currently `MapStore<string,any>`.  having a `Baggage` type would probably be better, but the point is that as far as baggage is concerned, keys are strings, and the value types are only known at the point of access. The Baggages are passed through many intermediaries, who don't and can't know what's being stored at higher and lower levels.

Reply:
Ah, right. It's not any particular baggage until it's what came from storage is validated to match a pattern. Would it make sense to validate the baggage data before passing it to the likes of `provideDurableIssuerKit`? If so I would propose `type Baggage<T=Record<string, unknown>> ` that we use for now and then when we have some data validation we can narrow to a type that has specific keys and value types.

Reply:
@turadg I don't understand the role of `Record` in the last suggestion, and thus I don't understand the last suggestion. We defined a `Baggage` type as `MapStore<string, unknown>` and then used it everywhere we declare baggage. So done enough? Resolving.